glitter/reminders/forms.py

```diff
@@ -0,0 +1,18 @@
+from django import forms
+
+from .models import Reminder
+
+
+class ReminderInlineAdminForm(forms.ModelForm):
+    class Meta:
+        model = Reminder
+        fields = ('user', 'interval',)
+
+    def clean_user(self):
+        user = self.cleaned_data['user']
+        if not user.email:
+            raise forms.ValidationError(
+                "{} - don't have email address please pick different user or enter "
```

Comment:
Would suggest changing this to:

`User doesn't have an email address, please pick a different user or add an email address`

To make it a bit more generic/readable.